src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoUnitTest.java

```diff
@@ -986,6 +994,78 @@ public void testCreateSnapshotWithProvidedIdsWithInvalidRowIds() throws Interrup
                 + ") AS T");
   }
 
+  @Test
+  public void aggregateSnapshotTableTest() {
+    String stringTest = "hello";
+    int intTest = 1234567;
+    List<String> listTest = List.of("a", "b", "c");
+
+    List<FieldValueList> listOfFieldValueList =
+        List.of(
+            FieldValueList.of(
+                List.of(
+                    FieldValue.of(FieldValue.Attribute.PRIMITIVE, stringTest),
+                    FieldValue.of(FieldValue.Attribute.PRIMITIVE, intTest),
+                    FieldValue.of(
+                        FieldValue.Attribute.REPEATED,
+                        listTest.stream()
+                            .map(s -> FieldValue.of(FieldValue.Attribute.PRIMITIVE, s))
+                            .collect(Collectors.toList())))));
+
+    List<StandardSQLTypeName> standardSQLTypeNames =
+        List.of(StandardSQLTypeName.STRING, StandardSQLTypeName.INT64, StandardSQLTypeName.ARRAY);
+
+    Schema schema =
+        Schema.of(
+            FieldList.of(
+                standardSQLTypeNames.stream()
+                    .map(
+                        s -> {
+                          Field.Builder fieldBuilder = Field.newBuilder(s.name(), s);
+                          if (s == StandardSQLTypeName.ARRAY) {
+                            fieldBuilder.setType(StandardSQLTypeName.STRING);
+                          }
+                          return fieldBuilder.build();
+                        })
+                    .collect(Collectors.toList())));
+
+    Page<FieldValueList> page =
+        new Page<>() {
+          @Override
+          public boolean hasNextPage() {
+            return false;
+          }
+
+          @Override
+          public String getNextPageToken() {
+            return "";
+          }
+
+          @Override
+          public Page<FieldValueList> getNextPage() {
+            return this;
+          }
+
+          @Override
+          public Iterable<FieldValueList> iterateAll() {
+            return listOfFieldValueList;
+          }
+
+          @Override
+          public Iterable<FieldValueList> getValues() {
+            return listOfFieldValueList;
+          }
+        };
+
+    TableResult table = new TableResult(schema, 10, page);
+
+    List<Map<String, Object>> result = BigQueryPdao.aggregateSnapshotTable(table);
+
+    assertEquals(result.get(0).get("STRING"), stringTest);
+    assertEquals(result.get(0).get("INT64"), intTest);
+    assertEquals(result.get(0).get("ARRAY"), listTest);
```

Comment:
I think these are reversed, the expected value comes first.
```suggestion
    assertEquals(stringTest, result.get(0).get("STRING"));
    assertEquals(intTest, result.get(0).get("INT64"));
    assertEquals(listTest, result.get(0).get("ARRAY"));
```